tools/ManifestSignerClient/ManifestSignerClient/CertificateUtil.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Cryptography;
+using Org.Webpki.JsonCanonicalizer;
```

Comment:
Is the JsonCanonicalizer a dll library? I noticed that we keep some of the source code in the project as well.

Reply:
If the source code have to be cloned, let's separate the JsonCanonicalizer as a separate project and reference it.

Reply:
You can just follow what https://github.com/cyberphone/json-canonicalization/tree/master/dotnet did in the solution by separating the canonicalizer and the es6numberserializer as reference projects.